GNUmakefile

```diff
@@ -142,6 +141,7 @@ SHADERS := $(SHADERS) \
 	src/gfx/vk/shaders/vsui.h
 
 DEFS := $(DEFS) \
+	-D_POSIX_C_SOURCE=200112L \
```

Comment:
Should the define also be added when building Android and WASM?

Reply:
I think I would leave it out for the other platforms unless it won't compile without it.

Reply:
So, you tested this, if it compiles correctly on the other platforms?

Reply:
On Mac and Linux yes, but I have not built Android or web before.  In theory those platforms shouldn't be doing any legacy posix stuff so it shouldn't matter if it's defined there, but it would probably a good idea for me to try it just in case.

Reply:
The other platforms appear to build with this change.  Android still requires it, but it was (and still is) defined in the Android specific makefile (android.mk) rather than needing it to be in the main makefile.